arlo-client/package.json

```diff
@@ -7,6 +7,7 @@
     "@blueprintjs/core": "^3.20.0",
     "@blueprintjs/table": "^3.8.1",
     "@testing-library/jest-dom": "^4.0.0",
+    "@testing-library/react": "^9.3.2",
```

Comment:
The description says:

> Moving `asyncActRender` from utilities to testUtilities so that `@testing-library/react` can stay in devDependencies.

But this is moving it to `dependencies`.

Reply:
@eventualbuddha Yeah, I initially thought that would resolve it on its own, but evidently the build still required it in the regular dependencies. I updated the comment to reflect that.